### ISSUE TITLE
Disable mask when certain characters entered

### ISSUE THREAD

**Issue by newmangreen** [RobinHerbots/Inputmask#1550]
Hi,
I have such a case that I couldn't find an answer. 
In our report parameter pages, we enable user to enter filter criteria in (masked) inputs like below;

\*.2017 : Means all 2017 records
\>1.000 : Means bigger than 1000
0506*60 : Means starts with 0506 and ends with 60

Users can also enter exact values without any filter characters (['*' , '!' , '>' , '<' , '=' , ';'])
What I want is to disable mask when these characters entered and re-enable it when these characters removed.
Thank you.

**Comment by RobinHerbots** [comment]
@newmangreen , 

Did you try with alternators?  Removing can also mean just using a mask which accepts everything

so something like (99-aaa-99)|(*{*})  This case will show the first mask as long the input for it is valid, otherwise will revert to  the other.

Make a jsfiddle to make it more concrete.

**Comment by newmangreen** [comment]
[https://jsfiddle.net/newmangreen/6unug1y4/](https://jsfiddle.net/newmangreen/6unug1y4/)

Alternators didn't help since;

- I couldn't use custom alias within alternator definiton
- If all characters entered complying with first mask, no key entry allowed to bypass first mask.

I developed a custom solution by listening keydown events and toggling mask.

One out-of-scope problem is that when removing inputmask, value is reverted to unmasked value although I used "autoUnmask:false" option.  See FIXME in jsfiddle.
